mapstring.c

```diff
@@ -2103,22 +2103,23 @@ int msStringIsInteger(const char *string)
 
 /* Safe version of msStrdup(). This function is taken from gdal/cpl. */
 
-char *msStrdup( const char * pszString )
+char *msStrdup(const char * pszString)
 {
-  char        *pszReturn;
 
-  if( pszString == NULL )
-    pszString = "";
+    if (pszString == NULL)
+        pszString = "";
 
-  pszReturn = strdup( pszString );
+    unsigned int nStringLength = strlen(pszString);
```

Comment:
Being in a .c file, the variable declarations should be done at the beginning of the function to please MSVC
```
size_t nStringLength;
char* pszReturn;
```